pkgs/applications/office/skrooge/default.nix

```diff
@@ -1,29 +1,34 @@
 { mkDerivation, lib, fetchurl,
-  cmake, extra-cmake-modules, qtwebkit, qtscript, grantlee,
+  cmake, extra-cmake-modules, qtwebengine, qtscript, grantlee,
   kxmlgui, kwallet, kparts, kdoctools, kjobwidgets, kdesignerplugin,
   kiconthemes, knewstuff, sqlcipher, qca-qt5, kactivities, karchive,
   kguiaddons, knotifyconfig, krunner, kwindowsystem, libofx, shared-mime-info
 }:
 
 mkDerivation rec {
   name = "skrooge-${version}";
-  version = "2.10.5";
+  version = "2.11.0";
 
   src = fetchurl {
     url = "http://download.kde.org/stable/skrooge/${name}.tar.xz";
-    sha256 = "1c1yihypb6qgbzfcrw4ylqr9zivyba10xzvibrmfkrilxi6i582n";
+    sha256 = "11ns0j3ss09aqd8snfzd52xf0cgsjjcgzalb031p7v17rn14yqaq";
   };
 
   nativeBuildInputs = [
     cmake extra-cmake-modules kdoctools shared-mime-info
   ];
 
   buildInputs = [
-    qtwebkit qtscript grantlee kxmlgui kwallet kparts
+    qtwebengine qtscript grantlee kxmlgui kwallet kparts
     kjobwidgets kdesignerplugin kiconthemes knewstuff sqlcipher qca-qt5
     kactivities karchive kguiaddons knotifyconfig krunner kwindowsystem libofx
   ];
 
+  cmakeFlags = [
+    "-DSKG_DESIGNER=OFF"
```

Comment:
Could you add a comment why this is turned off?

Reply:
I did so, hope everything is clear.